### ISSUE TITLE
Feature: Add web viewer for logs and other output

### ISSUE THREAD

**Issue by linknum23** [micro-nova/AmpliPi#256]
It would be very helpful to have an easy way have a live view of the output of amplipi and each of the streaming interfaces. 

Here's some things that we would need to support this

- [x] Backend log aggregation
- [x] SSE API endpoint
- [ ] Log viewer - Use the updater's log view as a template, needs to be on a separate web page to avoid amplipi's refreshes

**Comment by linknum23** [comment]
We investigated this today and found a collection of tools that will handle the SSE and backend using some journald.

Here's what we did on amplipi13:

    sudo apt install systemd-journal-remote
    sudo systemctl start systemd-journal-gatewayd.socket

Then to check SSE logs are working:

    curl --silent -H'Accept: text/even-stream' -H'Range: entries=:-80:' 192.168.0.152:19531/entries?follow

That command gets the last 80 logged events and continues sending all new log entries until the connection is closed. The underlying service does not appear to have a noticeable load.

The entries can also be viewed as a webpage by navigating to: http://amplipi13.local:19531/browse. This would be a candidate for log viewing if it updated automatically with new log messages. Instead we will probably use the update's log as a template for this.
